ssc/cmod_battery.cpp

```diff
@@ -91,11 +91,14 @@ var_info vtab_battery_inputs[] = {
         { SSC_INPUT,        SSC_NUMBER,      "batt_Vnom",                                  "Cell voltage at end of nominal zone",                     "V",       "",                     "BatteryCell",       "",                           "",                              "" },
         { SSC_INPUT,        SSC_NUMBER,      "batt_Vnom_default",                          "Default nominal cell voltage",                            "V",       "",                     "BatteryCell",       "",                           "",                              "" },
         { SSC_INPUT,        SSC_NUMBER,      "batt_Qfull",                                 "Fully charged cell capacity",                             "Ah",      "",                     "BatteryCell",       "",                           "",                              "" },
+        { SSC_INPUT,        SSC_NUMBER,      "batt_Qfull_cutoff",                                 "Fully charged cell capacity",                             "Ah",      "",                     "BatteryCell",       "",                           "",                              "" },
```

Comment:
Can you change the description to explain the difference between Qfull and Qfull_cutoff?

Reply:
We should talk about whether this is computed in SSC or SAM. It might be easier to maintain consistency between batt stateful and cmod battery if SSC does the computation.